src/tests/actions/userActions.test.js

```diff
@@ -1,11 +1,16 @@
 import MockAdapter from 'axios-mock-adapter';
+import thunk from 'redux-thunk';
 import configureMockStore from 'redux-mock-store';
-import { fetchUsers, loginUser } from '../../actions/userActions';
+import {
```

Comment:
I really liked that used one import per line from line `7-13` and in other files. Consider refactoring these please.

Reply:
I have modified this file to have one import per line